src/stories/components/CuTableColumnTeamMember/CuTableColumnTeamMember.tsx

```diff
@@ -77,44 +73,59 @@ const Container = styled.div({
   '@media (min-width: 834px) and (max-width: 1194px)': {
     marginLeft: '0px',
   },
-  [lightTheme.breakpoints.up('desktop_1194')]: {
+  [theme.breakpoints.up('desktop_1194')]: {
     marginRight: 36,
 
     alignItems: 'center',
   },
-  [lightTheme.breakpoints.up('desktop_1440')]: {
+  [theme.breakpoints.up('desktop_1440')]: {
     width: '100%',
     flex: 1,
     justifyContent: 'center',
   },
-  [lightTheme.breakpoints.up('desktop_1920')]: {
+  [theme.breakpoints.up('desktop_1920')]: {
     paddingLeft: 6,
     marginLeft: 0,
   },
-});
+}));
 
-const Data = styled.div({
+const Data = styled('div')(({ theme }) => ({
   display: 'flex',
   flexDirection: 'column',
   alignItems: 'center',
   marginRight: '10px',
-  [lightTheme.breakpoints.up('desktop_1194')]: {
+  [theme.breakpoints.up('desktop_1194')]: {
     marginRight: '10px',
   },
-  [lightTheme.breakpoints.up('desktop_1440')]: {
+  [theme.breakpoints.up('desktop_1440')]: {
     marginRight: '10px',
     marginLeft: '4px',
   },
-});
+}));
 
-const CirclesWrapper = styled.div({
+const CirclesWrapper = styled('div')({
   display: 'flex',
 });
 
-const Value = styled.div<{ isLight: boolean }>(({ isLight }) => ({
+const Value = styled('div')(({ theme }) => ({
   fontFamily: 'Inter, sans-serif',
   fontWeight: 600,
   fontSize: '14px',
   lineHeight: '17px',
-  color: isLight ? '#231536' : '#EDEFFF',
+  color: theme.palette.isLight ? '#231536' : '#EDEFFF',
+}));
+
+const Title = styled('span')(({ theme }) => ({
+  fontSize: '11px',
+
+  color: theme.palette.isLight ? '#434358' : '#9FAFB9',
+  fontWeight: 400,
+  fontStyle: 'normal',
+  marginBottom: '8px',
+  lineHeight: '13px',
+  fontFamily: 'Inter, sans-serif',
+  whiteSpace: 'nowrap',
+  [theme.breakpoints.up('desktop_1194')]: {
```

Comment:
This is a legacy breakpoint, we should not use it anymore

Reply:
done